conans/client/generators/cmake_find_package_multi.py

```diff
@@ -0,0 +1,102 @@
+from conans.client.generators.cmake import DepsCppCmake
+from conans.client.generators.cmake_find_package import find_dependency_lines
+from conans.client.generators.cmake_find_package_common import find_libraries, \
+    generic_common_set_vars
+from conans.model import Generator
+
+
+class CMakeFindPackageMultiGenerator(Generator):
+    template = generic_common_set_vars + """
+    
+# Load the debug and release library finders
+get_filename_component(_DIR "${{CMAKE_CURRENT_LIST_FILE}}" PATH)
+file(GLOB CONFIG_FILES "${{_DIR}}/Find{name}-*.cmake")
+
+if(NOT ${{CMAKE_VERSION}} VERSION_LESS "3.0")
+    # Target approach
+    if(NOT TARGET {name}::{name})
+        add_library({name}::{name} INTERFACE IMPORTED)
+    endif()
+endif()
+
+foreach(f ${{CONFIG_FILES}})
+  include(${{f}})
+endforeach()
+
+include(SelectLibraryConfigurations)
+select_library_configurations({name})
+
+# CMake doesn't have a mechanism to select also the right include directories, assume we can use any of them or use
+# the variable _DEBUG or _RELEASE
+set({name}_INCLUDE_DIRS ${{{name}_INCLUDE_DIRS_RELEASE}} ${{{name}_INCLUDE_DIRS_DEBUG}})
```

Comment:
This seems a bit risky. Could we go with this generator to use only modern CMake and targets?

Reply:
Not sure. It works apparently OK without targets. We could recommend to use the _RELEASE or _DEBUG and not the aggregated one...